nix/overlays/checked-shell-script/checked-shell-script.nix

```diff
@@ -2,13 +2,15 @@
 # directly, or use the .bin attribute to get the script in a bin/ directory,
 # to be used in a path for example.
 { git
+, lib
 , runCommand
 , runtimeShell
 , shellcheck
 , stdenv
 , writeTextFile
 }:
-name: text:
+{ name, docs, inRootDir ? false }: text:
+# TODO: do something sensible with docs, e.g. provide automated --help
```

Comment:
If we add `--help`, we'd need a way to pass the `--help` to the underlying command, too. Something like `-- --help` or so. Many of the commands pass arguments through and I sometimes use it.

Reply:
Yes, that was my worry - if we print the docs when `--help` or `-h` are on the command line, how do we make sure that we don't prevent the commands we use to show their help texts? `--` is the right solution probably. Will have to experiment with it.